samples/scenarios-monitoring/autoscaling-rules.bicep

```diff
@@ -0,0 +1,51 @@
+param location string = resourceGroup().location
+param appPlanName string = '${uniqueString(resourceGroup().id)}asp'
+
+resource appServicePlan 'Microsoft.Web/serverfarms@2021-03-01' = {
+  name: appPlanName
+  location: location
+  sku: {
+    name: 'S1'
+    tier: 'Standard'
+    size: 'S1'
+    family: 'S'
+    capacity: 1
+  } 
+}
+
+resource scaleUpRule 'Microsoft.Insights/autoscalesettings@2021-05-01-preview' = {
```

Comment:
Technically this is scaling *out* rather than *up*... but I think Autoscale might actually use the wrong term, in which case let's not worry :)